src/Files/Services/Implementation/PreferencesSettingsService.cs

```diff
@@ -44,7 +44,7 @@ public bool AreSystemItemsHidden
         
         public bool ShowDotFiles
         {
-            get => Get(false);
+            get => Get(true);
```

Comment:
Is this the same as file explorer?

Reply:
Yep, explorer does not hide files starting with dot by default